subsys/net/Kconfig

```diff
@@ -18,8 +18,8 @@ if NET_BUF
 
 config NET_BUF_USER_DATA_SIZE
 	int "Size of user_data available in every network buffer"
-	default 4
-	range 4 65535 if BT
+	default 8
```

Comment:
What's the reason for changing the default for everyone, even those who don't use Bluetooth? I'd do two defaults, just like the range:
```
default 8 if BT
default 4
```

Reply:
Right, this was intended just to BT actually